mw-webapp/src/component/editableText/EditableText.tsx

```diff
@@ -72,16 +73,29 @@ export const EditableText = <T extends string | number>(props: EditableTextProps
    * Check type of coming value and convert it to Number if need to use input with type "number"
    */
   const setValue = (value: string) => {
-    const number = Number(value) ?? 0;
-    const updatedValue = props.type === "number" ? number : value;
-    setText(updatedValue as T);
+    const checkedValue = value || 0;
```

Comment:
this looks a little strange, check it

Reply:
This is a check to ensure that the state does not contain an empty string.